src/parcsr_ls/HYPRE_parcsr_ls.h

```diff
@@ -4508,6 +4536,58 @@ HYPRE_ILUSolve( HYPRE_Solver solver,
 HYPRE_Int
 HYPRE_ILUSetMaxIter( HYPRE_Solver solver, HYPRE_Int max_iter );
 
+/**
+ * (Optional) Set the algorithm type for compute the ILU factorization. Options are:
```

Comment:
Typo. "to compute the iterative ILU ..."

Reply:
Fixed in https://github.com/hypre-space/hypre/pull/1011/commits/0ce97929ef0248becd67bf7551d2485937c701ed